src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,125 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.db.Manufacturer;
+import mate.jdbc.exeption.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.util.connection.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacture) {
+        String createRearmament = "INSERT INTO manufacturers(name, country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createFormatsStatement =
+                        connection.prepareStatement(createRearmament,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createFormatsStatement.setString(1, manufacture.getName());
+            createFormatsStatement.setString(2, manufacture.getCountry());
+            createFormatsStatement.executeUpdate();
+            ResultSet generatedKeys = createFormatsStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacture.setId(id);
+            }
+
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t put insert formats to db ", e);
+        }
+        return manufacture;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getRequest =
+                "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false";
+        Manufacturer manufacturer = new Manufacturer();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getFormatsStatement =
+                        connection.prepareStatement(getRequest)) {
+            getFormatsStatement.setLong(1,id);
+            ResultSet resultSet = getFormatsStatement.executeQuery();
+            if (resultSet.next()) {
+                manufacturer = getManufacture(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get formats from db ", e);
+        }
+        return Optional.ofNullable(manufacturer);
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> formats = new ArrayList<>();
+        String getAllReuarments = "SELECT * FROM manufacturers where is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllFormatsStatement = connection.createStatement()) {
```

Comment:
```suggestion
                PreparedStatement getAllFormatsStatement = connection.createStatement()) {
```
![image](https://user-images.githubusercontent.com/81781479/192294425-5be8c810-99ec-4948-9616-da58eed497f1.png)